### ISSUE TITLE
'Check for Updates' menu item in the desktop app fails to do anything

### ISSUE THREAD

**Issue by zillazillazilla** [Nault/Nault#60]
I am unsure if this is new to Nault or is an inherited issue from Nano Vault.

**Comment by Joohansson** [comment]
I tried to implement this:
https://medium.com/@johndyer24/creating-and-deploying-an-auto-updating-electron-app-for-mac-and-windows-using-electron-builder-6a3982c0cee6

But for some reason, the app completely ignores the code, so I gave up.

The check for update currently works for me on win10 in that sense it displays a notification:
![Capture](https://user-images.githubusercontent.com/2406720/90819488-54de1f80-e330-11ea-8aff-35526e489c21.PNG)

But nothing is downloaded or installed so it's mostly just misleading. I suggest we remove this feature until it can function. That is from the menu and from checking when the app is started.
